src/assets/icons/Delete.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="18" viewBox="0 0 14 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you use material icons for any of these icons?

Reply:
Why not use the icons from Figma?

Reply:
Consistency would be best. Custom icons are a special case but all these icons are available on through `material-icons`

Reply:
We should talk about using a different icon system at some point though. `react-feather` is pretty nice. Ideally we could extend a library in some way and add our own but have the same import and usage

Reply:
Alright, gonna go back to Material